en/views/json-and-xml-views.rst

```diff
@@ -70,6 +70,8 @@ serialize::
             $this->set('articles', $this->paginate());
             // Specify which view vars JsonView should serialize.
             $this->viewBuilder()->setOption('serialize', 'articles');
+            // Change the render mode
+            $this->RequestHandler->renderAs($this, 'json');
```

Comment:
This shouldn't be required unless you're not using extension routes or content-type parsing. I would prefer the docs recommend one of those setups vs doing this.

Reply:
For this section, it `lets you skip defining template files for your controller actions..`. 

On newly-installed Cake4, without calling `renderAs`, it will search for a template.

Reply:
This is all premised on how the request handler component works, that is it switches the view class depending on the URL extension or the `Accept` header. That example is ment to show a method that can render HTML as well as JSON, where HTML is the default.

An example on how to manually _force_ specific views can be found in the "Example Usage" section. This requires to _not_ use the request handler component, as it would be able to overwrite what your `renderAs()` call has configured.